osbrain/proxy.py

```diff
@@ -100,6 +100,30 @@ def _ready_or_timeout(self, time0, timeout):
             raise
         return True
 
+    def wait_for_running(self, timeout=0):
```

Comment:
Even though the type is well documented, I would rather use `0.` here. The default type can also help understanding which is the expected type for a parameter. :wink: